src/pages/console/data/content-management.mdx

```diff
@@ -68,8 +68,8 @@ You can also edit and store the data as markdown with the markdown editor. This
 ![cms](/images/console/cms.png)
 
 ## Limitations
-Amplify Studio's content management view requires your app to be using [DataStore](/lib/datastore/getting-started). 
+Amplify Studio's content management view requires your app's GraphQL API to have **Conflict Resolution** enabled. Conflict Resolution is used by [DataStore](/lib/datastore/getting-started) to handle data conflicts that arise from offline use. 
```

Comment:
Do you also want to add the IAM Auth mode as well? That REALLY tripped me up when building my last example app.

Reply:
Not for this update - I will add this change in a separate PR, as I need to iron out the specifics with the engineering team.